README.md

```diff
@@ -22,6 +22,11 @@ You can quickly setup a sample heroku hackmd application by clicking the button
 
 [![Deploy](https://www.herokucdn.com/deploy/button.svg)](https://heroku.com/deploy)
 
+You have to setup the minimal configuration
+
+- App Name ( Ex: hackmd )
+- HMD_DOMAIN ( Ex: hackmd.herokuapp.com )
```

Comment:
I do prefer to let Heroku handle domain itself. I'm not setting `HMD_DOMAIN` value, and both heroku domain(xxx.herokuapp.com) and my custom domain work. 😄